pkg/asset/installconfig/azure/validation.go

```diff
@@ -162,11 +162,21 @@ func validateFamily(fieldPath *field.Path, instanceType, family string) field.Er
 	windowsVMFamilies := sets.NewString(
 		"standardNVSv4Family",
 	)
+	diskNVMeVMFamilies := sets.NewString(
```

Comment:
```suggestion
	unsupportedDiskNVMeVMFamilies := sets.NewString(
```
I would suggest calling it unsupportedDiskNVMeVMFamilies

Reply:
I understand but then it won't match the existing pattern (e.g, `windowsVMFamilies`), so I'd rather not change that.